src/CsharpClient/QuixStreams.Streaming.UnitTests/StreamingClientShould.cs

```diff
@@ -0,0 +1,178 @@
+    }
+
+    private string topics = @"
+[
+    {
+        ""id"": ""confluent-testTopic"",
+        ""name"": ""confluent-testTopic"",
+        ""workspaceId"": ""confluent"",
+        ""status"": ""Ready"",
+    },
+    {
+        ""id"": ""quixdev-secondTest"",
+        ""name"": ""quixdev-secondTest"",
+        ""workspaceId"": ""quixdev"",
+        ""status"": ""Ready"",
+    },
+    {
+        ""id"": ""different-topic"",
+        ""name"": ""different-topic"",
+        ""workspaceId"": ""different"",
+        ""status"": ""Ready"",
+    }
+]";
+
+    private string workspaces = @"
+[
+    {
+        ""workspaceId"": ""confluent"",
+        ""name"": ""ConfluentCloud example"",
+        ""status"": ""Ready"",
+        ""brokerType"": ""ConfluentCloud"",
+        ""broker"": {
+            ""address"": ""xxxxx:9092"",
+            ""securityMode"": ""SaslSsl"",
+            ""sslPassword"": """",
+            ""saslMechanism"": ""Plain"",
+            ""username"": ""xxxxx"",
+            ""password"": ""xxxxx"",
+            ""hasCertificate"": false
+        },
+        ""workspaceClassId"": ""Standard"",
+        ""storageClassId"": ""Standard"",
+        ""createdAt"": ""2023-07-04T15:20:04.58Z"",
+        ""brokerSettings"": {
+            ""brokerType"": ""ConfluentCloud"",
+            ""syncTopics"": false,
+            ""confluentCloudSettings"": {
+                ""apiKey"": ""xxxxx"",
+                ""apiSecret"": ""xxxxx"",
+                ""clusterID"": ""xxxxx"",
+                ""bootstrapServer"": ""xxxxx:9092"",
+                ""restEndpoint"": ""xxxxx"",
+                ""clientID"": ""testclientid""
+            }
+        },
+        ""repositoryId"": ""xxxxx"",
+        ""branch"": ""main"",
+        ""environmentName"": ""prod"",
+        ""version"": 2,
+        ""branchProtected"": true
+    },
+    {
+        ""workspaceId"": ""quixdev"",
+        ""name"": ""Shared Kafka Workspace"",
+        ""status"": ""Ready"",
+        ""brokerType"": ""SharedKafka"",
+        ""broker"": {
+            ""address"": ""xxxx:9092"",
+            ""securityMode"": ""SaslSsl"",
+            ""sslPassword"": ""xxxx"",
+            ""saslMechanism"": ""ScramSha256"",
+            ""username"": ""xxxx"",
+            ""password"": ""xxxx"",
+            ""hasCertificate"": false
+        },
+        ""workspaceClassId"": ""Standard"",
+        ""storageClassId"": ""Standard"",
+        ""createdAt"": ""2023-07-07T14:02:00.932Z"",
+        ""brokerSettings"": {
+            ""brokerType"": ""SharedKafka"",
+            ""syncTopics"": false
+        },
+        ""repositoryId"": ""xxxxxxxxxx"",
+        ""branch"": ""chris"",
+        ""environmentName"": ""Testing"",
+        ""version"": 2,
+        ""branchProtected"": false
+    },
+    {
+        ""workspaceId"": ""different"",
+        ""name"": ""Unknown Kafka Workspace"",
+        ""status"": ""Ready"",
+        ""brokerType"": ""SharedKafka"",
```

Comment:
Why do we have this & brokerSettings.brokerType ?

Is this an error here or in the API?

Reply:
This is in the API. I think it is there because frontend is using it to show which kind of kafka we are using (and kept for compatibility of old workspaces)